packages/docs/src/components/features/ColorPalette.vue

```diff
@@ -66,54 +66,39 @@
   </section>
 </template>
 
-<script>
+<script setup lang="ts">
   // Utilities
   import kebabCase from 'lodash/kebabCase'
-  import colors from 'vuetify/lib/util/colors'
+  import { colors as vuetifyColors } from 'vuetify/lib/util/colors'
+  import { computed } from 'vue'
 
-  export default {
-    name: 'ColorPalette',
+  type Colors = Record<string, Record<string, string>>
 
-    data: () => ({
-      colors,
-      search: '',
-    }),
+  const colors: Colors = vuetifyColors
+  const search = ""
```

Comment:
Search now is a static assignment without any reactivity (also, since its a string primitive declared as const, is also an immutable value).

```suggestion
  const search = ref("")
```

I would suggest some further reading about [Vue Reactivity Fundamentals](https://vuejs.org/guide/essentials/reactivity-fundamentals.html)